### DISCUSSION TITLE
Some doubts

### DISCUSSION THREAD

**Discussion by bybpow** [audiamus/BookLibConnect#85] [Q&A] [answered]
- The final m4b is already decrypted?
- The exported aax is also decrypted to use in the converter?
- Do I need to keep the jsons for the converter?

Thanks in advance.

**Comment by audiamus** [comment, marked as answer]
Quote from the manual, chapter 3.3:

> Export is an option for compatibility with current AAX Audio Converter (version 1.*). The unlocked
books will be in plain M4B format (MP4 audiobook) and can be played with a vast variety of
players. You can further process them with AAX Audio Converter, to convert to MP3, apply splitting
or alter meta data.
For this the .m4b files are simply copied as file type .aax, and additional meta data is extracted from
the local database and saved as .json files, for chapter names and series information. AAX Audio
Converter will use such meta data if available.

**Reply by bybpow** [reply]
Thanks.
